### PR TITLE
Drop `cython` from run requirements

### PR DESCRIPTION
**Pull request by jakirkham**
As `cython` is only needed to build the Python bindings and does not contain any needed runtime libraries, drop it from the run requirements.

### CODE REVIEW

**Comment by jakirkham** [comment]
rerun tests

File: CHANGELOG.md [hunk 1]
```diff
@@ -18,6 +18,7 @@
 - PR #32 Python API first pass
 - PR #37 Python __init__.py package design
 - PR #38 Add __init__.py empties to resolve issue with PYTHONPATH
+- PR #97 Drop `cython` from run requirements
```
Comment (harrism) [inline CHANGELOG.md]: This should be under bug fixes or improvements. It's not a new feature.

Comment (jakirkham) [reply CHANGELOG.md]: Ah thought it was under improvements. Sorry. Will move. Thanks.

**Comment by jakirkham** [comment]
rerun tests

**Comment by jakirkham** [comment]
Thanks @kkraus14! 😄

**Pull request merged**